src/metabase/api/automagic_dashboards.clj

```diff
@@ -101,6 +101,12 @@
   [_entity-type segment-id-str]
   (api/read-check (db/select-one Segment :id (ensure-int segment-id-str))))
 
```

Comment:
Add the `/auto/dashboard/model/:id` route. Note that this will ensure the requested entity is a model, where `/auto/dashboard/question/:id` will not. The latter will still product a model x-ray for a model, but will do something else for questions.